src/window.rs

```diff
@@ -182,6 +182,15 @@ impl Window {
         self.window.set_inner_size(size);
     }
 
+    // TODO: use `with_position` once available
+    pub fn set_position(&mut self, x: i32, y: i32) {
+        let dpr = self.window.get_hidpi_factor();
+        let physical = PhysicalPosition::from((x, y));
+        let logical = physical.to_logical(dpr);
```

Comment:
This is a tiny nitpick, but I think it's a bit too verbose (too many variables that aren't reused, but just temporary).

Something like this should work a bit nicer:
```rust
let logical = PysicalPosition::from((x, y)).to_logical(self.window.get_hidpi_factor());
self.window.set_position(logical);
```

I believe this would still be under the 100 column limit, so it shouldn't be a problem. Alternatively just declaring a separate variable for `dpr` would be enough and we could at least drop the `physical` variable.